frontends/qt/server/server.go

```diff
@@ -76,11 +80,23 @@ func serve(
 	responseCallback C.responseCallback,
 	notifyUserCallback C.notifyUserCallback,
 ) {
+	log := logging.Get().WithGroup("server")
+	log.WithField("args", os.Args).Info("Started Qt application")
 	// workaround: this flag is parsed by qtwebengine, but flag.Parse() quits the app on
 	// unrecognized flags
 	// _ = flag.Int("remote-debugging-port", 0, "")
 	testnet := flag.Bool("testnet", false, "activate testnets")
 
+	if runtime.GOOS == "darwin" {
+		// eat "-psn_xxxx" on Mac, which is passed when starting an app from Finder for the first time.
+		// See also: https://stackoverflow.com/questions/10242115/os-x-strange-psn-command-line-parameter-when-launched-from-finder
+		for _, arg := range os.Args[1:] {
+			if strings.HasPrefix(arg, "-psn_") {
+				_ = flag.Bool(arg[1:], false, "<ignored>")
```

Comment:
@x1ddos any idea how to ignore an argument more nicely?

Reply:
Not really. This is kind of the "canonical" way to ignore flags with stdlib flag package, for better or worse... :)
A slightly nicer way would be to get rid of `_ =` assignment part. :-p

Also, I would probably do something like `strings.TrimLeft(arg, "-")` as a tiny bit more future proof in case they start using double dashes in the flags.

Reply:
Thx :pray: